UI/EnergyLabel.gd

```diff
@@ -0,0 +1,20 @@
+extends Label
+
+var energy_component: EnergyComponent
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
```

Comment:
All the functions in this file are missing a return type (which causes errors when loading the game, since missing typing is treated as an error now)